Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -274,8 +274,11 @@ private class SyncNowSetting: WithAccountSetting {
             return nil
         }
 
-        let label = NSLocalizedString("Last synced: %@", comment: "Last synced time label beside Sync Now setting option. Argument is the relative date string.")
-        let formattedLabel = String(format: label, NSDate.fromTimestamp(timestamp).toRelativeTimeString())
+        let formatter = NSDateFormatter();
+        formatter.dateFormat = "yyyy-MM-dd HH:mm:ss";
+
+        let label = NSLocalizedString("%@", comment: "Last synced time label beside Sync Now setting option. Argument is the relative date string.")
```

Comment:
If we're specifying a fixed date format — i.e., not one that's localized — then this doesn't need to be a localized string.

And we are, so it doesn't. Just:

```
let formattedLabel = formatter.stringFromDate(NSDate.fromTimestamp(timestamp))
```